disk.go

```diff
@@ -0,0 +1,608 @@
+package disko
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/apex/log"
```

Comment:
Using logging in this context is probably the same as using fmt.Printf as we wont log to files. Can we do without logging for library.

Reply:
I'll drop the logging for now.